packages/frontend-acceptance-tests/src/features/8 - Recurring Payments/1_adult_licence-12-months_immediate_start_recurring.feature

```diff
@@ -0,0 +1,150 @@
+@browser
+Feature: I want to buy a recurring adult annual fishing
+
+  Background: Buy an adult fishing licence
+    Given  I am at the start of the purchase journey
+
+  Scenario: Scenario 1 - 12 Month Adult licence selecting 2 rod sea trout licence - Immediate start - Enter contact details
+    And I am buying a licence for myself
+    And I enter "Adult" "Salmon" as the name
+    And I am 7 days over my 17th birthday
```

Comment:
Are 17 year olds allowed to set up a recurring payment?

Reply:
I based this off function in gafl
export const validForRecurringPayment = permission => {
  const licenseeAge = moment().tz(SERVICE_LOCAL_TIME).diff(moment(permission.licensee.birthDate), 'years')
  return (
    process.env.SHOW_RECURRING_PAYMENTS?.toLowerCase() === 'true' &&
    permission.licenceLength === '12M' &&
    permission.isLicenceForYou &&
    licenseeAge > JUNIOR_MAX_AGE &&
    process.env.CHANNEL?.toLowerCase() !== 'telesales'
  )
}
JUNIOR_MAX_AGE = 16

Reply:
I'm not sure this is a correct assumption; we charge for an adult licence in an angler's eighteenth year (i.e. it is on or after their seventeenth birthday). However, I'm not sure where we stand with offering recurring payments to someone who is, essentially, a minor, even if they won't be by the time the recurring payment comes into force. Let's pick this issue up with Melinda before we merge this PR.

Reply:
I've asked Melinda, although I'm unlikely to hear back before Wednesday I think

Reply:
Darren said we're in the clear on this, so we're ok to merge this PR now